sherpa/ui/tests/test_session.py

```diff
@@ -895,3 +895,80 @@ def test_paramprompt_eof(caplog):
     assert cpt2.ampl.val == pytest.approx(1)
     assert cpt2.ampl.min < -3e38
     assert cpt2.ampl.max > 3e38
+
+
+def test_delete_model_component_invalid_argument():
+    """We could allow the component to be deleted this way"""
```

Comment:
I think we should. Could be in the PR, but probably better goes into a new one, so we might want to open an issue?

Reply:
Done - #1357